mitmproxy/utils/emoji.py

```diff
@@ -0,0 +1,1808 @@
+    ":waning_crescent_moon:": "🌘",
+    ":waning_gibbous_moon:": "🌖",
+    ":warning:": "⚠",
+    ":wastebasket:": "🗑",
+    ":watch:": "⌚",
+    ":water_buffalo:": "🐃",
+    ":water_polo:": "🤽",
+    ":watermelon:": "🍉",
+    ":wave:": "👋",
+    ":wavy_dash:": "〰",
+    ":waxing_crescent_moon:": "🌒",
+    ":waxing_gibbous_moon:": "🌔",
+    ":wc:": "🚾",
+    ":weary:": "😩",
+    ":wedding:": "💒",
+    ":weight_lifting:": "🏋",
+    ":weight_lifting_man:": "🏋‍♂",
+    ":weight_lifting_woman:": "🏋‍♀",
+    ":western_sahara:": "🇪‍🇭",
+    ":whale:": "🐳",
+    ":whale2:": "🐋",
+    ":wheel_of_dharma:": "☸",
+    ":wheelchair:": "♿",
+    ":white_check_mark:": "✅",
+    ":white_circle:": "⚪",
+    ":white_flag:": "🏳",
+    ":white_flower:": "💮",
+    ":white_haired_man:": "👨‍🦳",
+    ":white_haired_woman:": "👩‍🦳",
+    ":white_heart:": "🤍",
+    ":white_large_square:": "⬜",
+    ":white_medium_small_square:": "◽",
+    ":white_medium_square:": "◻",
+    ":white_small_square:": "▫",
+    ":white_square_button:": "🔳",
+    ":wilted_flower:": "🥀",
+    ":wind_chime:": "🎐",
+    ":wind_face:": "🌬",
+    ":wine_glass:": "🍷",
+    ":wink:": "😉",
+    ":wolf:": "🐺",
+    ":woman:": "👩",
+    ":woman_artist:": "👩‍🎨",
+    ":woman_astronaut:": "👩‍🚀",
+    ":woman_cartwheeling:": "🤸‍♀",
+    ":woman_cook:": "👩‍🍳",
+    ":woman_dancing:": "💃",
+    ":woman_facepalming:": "🤦‍♀",
+    ":woman_factory_worker:": "👩‍🏭",
+    ":woman_farmer:": "👩‍🌾",
+    ":woman_firefighter:": "👩‍🚒",
+    ":woman_health_worker:": "👩‍⚕",
+    ":woman_in_manual_wheelchair:": "👩‍🦽",
+    ":woman_in_motorized_wheelchair:": "👩‍🦼",
+    ":woman_judge:": "👩‍⚖",
+    ":woman_juggling:": "🤹‍♀",
+    ":woman_mechanic:": "👩‍🔧",
+    ":woman_office_worker:": "👩‍💼",
+    ":woman_pilot:": "👩‍✈",
+    ":woman_playing_handball:": "🤾‍♀",
+    ":woman_playing_water_polo:": "🤽‍♀",
+    ":woman_scientist:": "👩‍🔬",
+    ":woman_shrugging:": "🤷‍♀",
+    ":woman_singer:": "👩‍🎤",
+    ":woman_student:": "👩‍🎓",
+    ":woman_teacher:": "👩‍🏫",
+    ":woman_technologist:": "👩‍💻",
+    ":woman_with_headscarf:": "🧕",
+    ":woman_with_probing_cane:": "👩‍🦯",
+    ":woman_with_turban:": "👳‍♀",
+    ":womans_clothes:": "👚",
+    ":womans_hat:": "👒",
+    ":women_wrestling:": "🤼‍♀",
+    ":womens:": "🚺",
+    ":woozy_face:": "🥴",
+    ":world_map:": "🗺",
+    ":worried:": "😟",
+    ":wrench:": "🔧",
+    ":wrestling:": "🤼",
+    ":writing_hand:": "✍",
+    ":x:": "❌",
+    ":yarn:": "🧶",
+    ":yawning_face:": "🥱",
+    ":yellow_circle:": "🟡",
+    ":yellow_heart:": "💛",
+    ":yellow_square:": "🟨",
+    ":yemen:": "🇾‍🇪",
+    ":yen:": "💴",
+    ":yin_yang:": "☯",
+    ":yo_yo:": "🪀",
+    ":yum:": "😋",
+    ":zambia:": "🇿‍🇲",
+    ":zany_face:": "🤪",
+    ":zap:": "⚡",
+    ":zebra:": "🦓",
+    ":zero:": "0‍⃣",
+    ":zimbabwe:": "🇿‍🇼",
+    ":zipper_mouth_face:": "🤐",
+    ":zombie:": "🧟",
+    ":zombie_man:": "🧟‍♂",
+    ":zombie_woman:": "🧟‍♀",
+    ":zzz:": "💤",
+}
+
+if __name__ == "__main__":
```

Comment:
You can add a  `# pragma: no cover` to the end of this line so that our coverage checks are not complaining. :)